### PR TITLE
Implement undo command

### PR DESCRIPTION
**Pull request by etlow**

### CODE REVIEW

**Comment by ryantay232** [comment]
Hope you can resolve the conflict! Good luck!

File: src/main/java/seedu/address/logic/commands/UndoCommand.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    public static final String COMMAND_WORD = "undo";
+
+    public static final String MESSAGE_SUCCESS = "Successfully undid: %1$s";
+    public static final String MESSAGE_NOTHING_TO_UNDO = "There is nothing to undo";
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
```
Comment (charliechoong) [inline src/main/java/seedu/address/logic/commands/UndoCommand.java]: Looks good

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 2]
```diff
@@ -15,6 +15,8 @@
 import seedu.address.logic.commands.FindCommand;
```
Comment (charliechoong) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java]: This will be useful for our tP.

**Review by MackyMaguire — COMMENTED** [review]
Fix the merge conflict and its good to merge.

File: src/test/java/seedu/address/logic/commands/UndoCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.person.Person;
+
+class UndoCommandTest {
```
Comment (ryantay232) [inline src/test/java/seedu/address/logic/commands/UndoCommandTest.java]: Great job making tests!

**Review by ryantay232 — COMMENTED** [review]
Looks good! Hope the conflict gets resolved.

**Pull request merged**